src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/OrganisationCreationRequestValidator.java

```diff
@@ -170,4 +174,14 @@ private void isDxAddressValid(DxAddressCreationRequest dxAddress) {
         }
     }
 
+    public static void isInputOrganisationStatusValid(String organisationStatus, String allowedStatus) {
+        List<String> validStatusList = asList(allowedStatus.split(","));
+        String orgStatus = removeAllSpaces(organisationStatus);
+        if (isBlank(orgStatus) ||  !validStatusList.contains(orgStatus.toUpperCase())) {
+            String errorMessage = "Please check status param passed as this is invalid status.";
```

Comment:
Move the message to ProfessionalApiConstants file and reuse

Reply:
Done